docs/generated/openapi.yaml

```diff
@@ -3667,6 +3859,39 @@ components:
         name:
           type: string
           example: east
+    DataplaneNetworkingLayout:
+      type: object
+      title: DataplaneLayout
+      description: >-
+        Dataplane networking layout. It contains information most important
+        information about dataplane and lists of available inbounds and
+        outbounds
+      required:
```

Comment:
should all fields be required? Otherwise we have these pointers for arrays and maps:
```go
type DataplaneNetworkingLayout struct {
	Inbounds  *[]externalRef0.DataplaneInbound  `json:"inbounds,omitempty"`
	Kri       string                            `json:"kri"`
	Labels    *map[string]string                `json:"labels,omitempty"`
	Name      string                            `json:"name"`
	Outbounds *[]externalRef0.DataplaneOutbound `json:"outbounds,omitempty"`
}
```
but we don't actually need to differentiate between unset labels and empty labels